`install.sh --rpackage` does not work properly

I found that `Installing --rpackage... OK` while it is not OK.
```
ggruszcz@rysy-n2 ~/rysy/TCLB $ tools/install.sh --rpackage buhahaha  # Honestly, I tried to install the 'png' package...
Discovered Package Manager: yum
Installing --rpackage... OK
```

My configuration:
```bash
ggruszcz@rysy-n2 ~/rysy/TCLB $ git fetch --all
Fetching origin
Fetching upstream
ggruszcz@rysy-n2 ~/rysy/TCLB $ git merge upstream/develop 
Already up-to-date.
ggruszcz@rysy-n2 ~/rysy/TCLB $ git branch -v
* develop                           f187227 [ahead 2] Merge pull request #329 from llaniewski/fix/grid3d
```

It works today! I don't know what had happened. I now charged the compute20 to compute60 in the make file. Hope it will not cause problems. 


base) ztdep@linux-9owh:~/Downloads/TCLB> make -j 20 d2q9
make: Circular CLB/d2q9/makefile <- CLB/d2q9/makefile dependency dropped.
  AUTO-DEP   CLB/d2q9/dep.mk
  RT         CLB/d2q9/makefile
make[1]: Entering directory '/home/ztdep/Downloads/TCLB/CLB/d2q9'
  G++        main.cpp
  NVCC       cuda.cu
  G++        vtkLattice.cpp
  G++        Solver.cpp
  NVCC       Lattice.cu
  NVCC       cross.cu
  LINKING    empty
  LINKING    compare
nvcc fatal   : Value 'compute_20' is not defined for option 'gpu-architecture'
make[1]: *** [makefile:63: cuda.o] Error 1
make[1]: *** Waiting for unfinished jobs....
  LINKING    simplepart
nvcc fatal   : Value 'compute_20' is not defined for option 'gpu-architecture'
make[1]: *** [makefile:63: cross.o] Error 1
nvcc fatal   : Value 'compute_20' is not defined for option 'gpu-architecture'

--- a/.travis.yml
+++ b/.travis.yml
@@ -50,7 +50,6 @@ before_install:
     - tools/install.sh rdep
     - tools/install.sh rinside
     - sudo tools/install.sh python-dev
-    - tools/install.sh rpython
     - tools/install.sh submodules
 
 install:

--- a/tools/install.sh
+++ b/tools/install.sh
@@ -275,7 +275,8 @@ do
 		;;
 	rpython)
 		install_rpackage rjson
-		install_rpackage rPython
+		echo "rPython not supported anymore"
+		# install_rpackage rPython
 		;;
 	rinside)
 		if $GITHUB

--- a/tools/install.sh
+++ b/tools/install.sh
@@ -74,7 +74,9 @@ function install_rpackage {
 			dir.create(p,recursive=TRUE);
 			.libPaths(p);
 		}
+		if (! "$name" %in% available.packages()[,1]) stop("$name not available on CRAN");
 		install.packages('$name', method="wget");
+		if (! require('$name')) stop("Failed to load $name");
 EOF
 }
 
@@ -245,6 +247,7 @@ do
 		#try "Changing access to R lib paths" chmod 2777 /usr/local/lib/R /usr/local/lib/R/site-library
 		;;
 	-r|--rpackage)
+		shift
 		test -z "$1" && error "usage tools/install.sh [--github] --rpackage package_name"
 		if $GITHUB
 		then
@@ -283,6 +286,7 @@ do
 		fi
 		;;
 	cuda)
+		shift
 		test -z "$1" && error "Version number needed for cuda install"
 		CUDA=$1
 		shift

